vaadin-login-flow/src/main/java/com/vaadin/flow/component/login/AbstractLogin.java

```diff
@@ -106,6 +106,25 @@ public boolean isError() {
         return getElement().getProperty("error", false);
     }
 
+    /**
+     * Sets whether to show or hide the forgot password button.
+     * The button is visible by default
+     *
+     * @see #isForgotPasswordButtonVisible()
+     */
+    public void setForgotPasswordVisible(boolean forgotPasswordVisible) {
+        getElement().setProperty("noForgotPassword", !forgotPasswordVisible);
+    }
+
+    /**
+     * Returns whether the forgot password button is visible or not
+     *
+     * @return {@code true} if the forgot password button is visible
+     *         {@code false} otherwise
+     */
+    public boolean isForgotPasswordButtonVisible() {
```

Comment:
This accessor's name is not consistent with the mutator's ( `setForgot[Button]PasswordVisible` )